### PR TITLE
v2.1.0

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$PCENGINES_DOWNLOAD \
+          -v dpp_logs_key:$PCENGINES_LOGS -v netboot_utilities_support:True \
+          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe
+          -i "pcengines" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_pcengines.log | grep "| PASS |\|| FAIL |"
+
+          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
+          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$PCENGINES_SEABIOS_DOWNLOAD \
+          -v dpp_logs_key:$PCENGINES_SEABIOS_LOGS -v netboot_utilities_support:True \
+          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe
+          -i "pcengines_seabios" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_pcengines_seabios.log | grep "| PASS |\|| FAIL |"
```
Comment (m-iwanicki) [inline .github/workflows/test.yml:113]:
```suggestion
          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$MSI_DOWNLOAD \
          -v dpp_logs_key:$MSI_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "msi" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_msi.log | grep "| PASS |\|| FAIL |"

          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$MSI_HEADS_DOWNLOAD \
          -v dpp_logs_key:$MSI_HEADS_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "msi_heads" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_msi_heads.log | grep "| PASS |\|| FAIL |"

          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$OPTIPLEX_DOWNLOAD \
          -v dpp_logs_key:$OPTIPLEX_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "optiplex" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_optiplex.log | grep "| PASS |\|| FAIL |"

          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$NOVACUSTOM_HEADS_DOWNLOAD \
          -v dpp_logs_key:$NOVACUSTOM_HEADS_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "novacustom_heads" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_nc_heads.log | grep "| PASS |\|| FAIL |"

          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$PCENGINES_DOWNLOAD \
          -v dpp_logs_key:$PCENGINES_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "pcengines" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_pcengines.log | grep "| PASS |\|| FAIL |"

          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
          -v dpp_password:$DPP_PASSWORD -v dpp_download_key:$PCENGINES_SEABIOS_DOWNLOAD \
          -v dpp_logs_key:$PCENGINES_SEABIOS_LOGS -v netboot_utilities_support:True \
          -v dts_ipxe_link:http://${ip_addr}:4321/dts.ipxe \
          -i "pcengines_seabios" dts/dts-e2e.robot 2>&1 | tee $LOG_DIR/output_pcengines_seabios.log | grep "| PASS |\|| FAIL |"
```

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (PLangowski) [reply .github/workflows/test.yml]: [8226575](https://github.com/Dasharo/meta-dts/pull/196/commits/822657590377da31fbdd4351a2299ab8085e0278)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,363 +1,956 @@
...
+## [1.0.1] - 2022-09-02
+
+### Changed
+- Initial commit
+- Initial public release
+- Fix README
+
+
+[unreleased]: https://github.com/Dasharo/meta-dts/compare/v1.2.23..HEAD
+[1.2.23]: https://github.com/Dasharo/meta-dts/compare/v1.2.22..v1.2.23
```
Comment (m-iwanicki) [inline CHANGELOG.md:921]: You are missing current `rc` links (`v2.0.0-rc*` & `v2.0.1-rc*`)

**Review by m-iwanicki — CHANGES_REQUESTED** [review]
Are you going to add `v2.1.0` to changelog?
And are we going to skip `v2.0.1`?

File: meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb [hunk 3]
```diff
@@ -6,6 +6,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
 
 DEPENDS = "pciutils libusb libftdi"
 
+PV = "dasharo-v1.2.2+git${SRCPV}"
```
Comment (m-iwanicki) [inline meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb:9]: This line results in:

```
WARNING: preferred version git of flashrom not available (for item flashrom)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
WARNING: preferred version git of flashrom not available (for item flashrom)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
WARNING: preferred version git of flashrom not available (for item flashrom-dev)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
```

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

**Comment by m-iwanicki** [comment]
There is something wrong with workflow or runner:

image

If you close/reopen PR or add new commit then either one of those jobs would start (and next one would probably wait forever again). At least that's what happened when I tried it.
As there is no separation in workflow workdir cleanup has to run before build or next job (`test`) would fail.

Workflows:
* https://github.com/Dasharo/meta-dts/actions/runs/12163220462
* https://github.com/Dasharo/meta-dts/actions/runs/12160278084

File: meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb (hunk 3 above)
Comment (PLangowski) [reply meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb:9]: https://github.com/Dasharo/meta-dts/pull/196/commits/5b6bd5a07f848b46904f0f5dfd50ad7046794d53

**Comment by PLangowski** [comment]
@m-iwanicki Can you check if everything is fine here? (aside from CI)

**Comment by m-iwanicki** [comment]
I did 2 weeks ago: https://github.com/Dasharo/meta-dts/pull/196#pullrequestreview-2464436794 and https://github.com/Dasharo/meta-dts/pull/196#discussion_r1860297594. Except changelog  everything else looks ok.

**Comment by m-iwanicki** [comment]
And check if this is correct srcrev and all changes are in changelog: https://github.com/Dasharo/meta-dts/pull/196/commits/629bc4cb64ea30f2d8d9308aaa9f30cd7bf03cb0

File: CHANGELOG.md [hunk 4]
```diff
@@ -917,7 +982,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix README
 
 
-[unreleased]: https://github.com/Dasharo/meta-dts/compare/v1.2.23..HEAD
+[2.1.0]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc3..v2.1.0
+[2.0.1-rc3]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc2..v2.0.1-rc3
+[2.0.1-rc2]: https://github.com/Dasharo/meta-dts/compare/v2.0.1-rc1..v2.0.1-rc2
+[2.0.1-rc1]: https://github.com/Dasharo/meta-dts/compare/v2.0.0..v2.0.1-rc1
+[2.0.0]: https://github.com/Dasharo/meta-dts/compare/v2.0.0-rc6..v2.0.0
```
Comment (m-iwanicki) [inline CHANGELOG.md:988]: Why from `rc6` and not `rc7`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,363 +1,1031 @@
...
+- .github/workflows/test.yml: Specify Python version (3.12)
+- .github/workflows/test.yml: Fix
+- .github/workflows/test.yml: Create hdd.qcow using `touch`
+- workflow: Fix build failure reported as success
+- dts-distro.conf: bump to 2.1.0
+- .github/workflows/test.yml: Fix job name in `needs`
+- .github/workflows/test.yml: Hotfix
+- .github/workflows/test.yml: Add missing backslashes
+- dts-scripts: Improve UI/UX
+- dts-scripts: Hotfix
```
Comment (m-iwanicki) [inline CHANGELOG.md:46]: Do we need so many commits about workflow in changelog?
And some commits e.g.:

* `dts-scripts: Hotfix`
* `.github/workflows/test.yml: Fix`

don't add anything as you can't deduce from changelog what was fixed/changed.

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (PLangowski) [reply CHANGELOG.md]: You're right. I removed some notes https://github.com/Dasharo/meta-dts/compare/8a35e7fc1219311ddbad68dbd7927ecda4322a5f..3ed77b24899849f939d8bf1a173027837e63bed4

**Comment by m-iwanicki** [comment]
@PLangowski Are we following release process from <https://docs.zarhus.com/development-process/standard-release-process/#process-steps>? If yes then what about `Use the generate-changed-recipes script`

**Comment by PLangowski** [comment]
@m-iwanicki since this will be the first release that publishes its manifest, I think we should use the script in the next release, where we will have 2 "official" manifests to compare.

File: CHANGELOG.md (hunk 4 above)
Comment (PLangowski) [reply CHANGELOG.md]: fixed https://github.com/Dasharo/meta-dts/compare/3ed77b24899849f939d8bf1a173027837e63bed4..0795508d026a009491115788cbc4fa0e560309a8

**Review by m-iwanicki — APPROVED** [review]
LGTM

**Pull request merged**